merge_data/healthcare_FR_finess.mapping.json

```diff
@@ -177,7 +189,57 @@
   },
   {
     "categories": [
-      "4501"
+      "202"
+    ],
+    "items": [
+      8340,
+      7230
+    ],
+    "missing_osm": false,
+    "classes": 100,
+    "level": 3,
+    "title": "Résidence autonomie",
+    "tags_select": {
+      "amenity": "social_facility"
+    },
+    "tags_generate1": {
+      "amenity": "social_facility",
+      "social_facility": "assisted_living",
+      "social_facility:for": "senior"
+    },
+    "tags_generate2": {}
+  },
+  {
+    "categories": [
+      "500"
+    ],
+    "items": [
+      8340,
+      7230
+    ],
+    "missing_osm": false,
+    "classes": 110,
+    "level": 3,
+    "title": "EHPAD",
+    "tags_select": {
+      "amenity": "social_facility"
+    },
+    "tags_generate1": {
+      "amenity": "social_facility",
+      "social_facility": "group_home",
```

Comment:
comment on fait pour remplacer ? là c'est un commentaire mais j'aimerai que group_home soit remplacé par nursing_home

Reply:
https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request
look for suggestion